tools/wa_user_directory/config.yaml

```diff
@@ -4,6 +4,25 @@ device: generic_android
 trace-cmd:
   buffer_size: 102400
   report: false
+  events: [
+    "sched_switch",
+    "sched_wakeup",
+    "sched_wakeup_new",
+    "sched_overutilized",
+    "sched_load_avg_cpu",
+    "sched_load_avg_task",
+    "sched_load_waking_task",
+    "cpu_capacity",
+    "cpu_frequency",
+    "cpu_idle",
+    "sched_tune_config",
+    "sched_tune_tasks_update",
+    "sched_tune_boostgroup_update",
+    "sched_tune_filter",
```

Comment:
What in case certain events are not available? Does WA still configure the remaining ones?

Reply:
You just get a warning like "the event xxx is not available" and all the other ones are collected properly.